shell/app/config-page/components/scale-card/scale-card.tsx

```diff
@@ -17,18 +17,22 @@ import React from 'react';
 import './scale-card.scss';
 
 const ScaleCard = (props: CP_SCALE_CARD.Props) => {
-  const { execOperation, data } = props;
+  const { execOperation, data, onClick, props: configProps } = props;
   const { list } = data || {};
+  const { align } = configProps || {};
```

Comment:
I think its a anti ts pattern behavior, type of `configProps` is not subset of `undefined`, then basically, no need add `|| {}`

Reply:
configProps is a subset of undefined